File: Latihan B/PYTHON/Processor.py
class Processor:

    # private atribut
    __name = ""
    __price = 0

    # konsturktor
    def __init__(self, name="", price=0):
        self.__name = name
        self.__price = price

    # set semua variable
    def setName(self, name):
        self.__name = name
    def setPrice(self, price):
        self.__price = price

    # get semua variabel
    def getName(self):
        return self.__name
    def getPrice(self):
        return self.__price
    
    # tampilan untuk Processor
    def printProc(self):
        print("  Nama Processor     : " + str(self.getName()))
        print("  Harga Processor    : " + str(self.getPrice()))

File: Latihan A/C++/Processor.cpp
#include <iostream>

class Processor{
    private:
        int price;
        string name;
    public:

        Processor(){}
        Processor(int price, string name){
            this->price = price;
            this->name = name;
        }

        //set semua variaber
        void setPricep(int price){
            this->price = price;
        }
        void setNamep(string name){
            this->name = name;
        }

        //get semua variabel
        int getPricep(){
            return this->price;
        }

        string getNamep(){
            return this->name;
        }

        //tampilan untuk prosesor
        void printProc(){
            cout <<"  Nama Processor     : " << this->name << endl;
            cout <<"  Harga Processor    : " << this->price << endl;
        }
        ~Processor(){
            
        }
};

